src/components/10-atoms/toggle-switch/index.scss

```diff
@@ -0,0 +1,60 @@
+$toggle-switch-animation: 0.4s ease;
+$toggle-switch-height: 30px;
+$toggle-switch-width: 50px;
+$toggle-switch-spacing: 2px;
+
+.a-toggle-switch {
+  display: block;
```

Comment:
With `block` its not possible to place other elements next to the switch. But this could be a possible workflow. Why not `inline-block`?